PointsRanking/PointsRankingCalc.php

```
@@ -0,0 +1,669 @@
+
+    return ['kept' => $kept, 'dropped' => $dropped, 'total' => array_sum($kept)];
+}
+
+/**
+ * Sort by points desc then place asc; equal points share a rank.
+ *
+ * @param array $rows Each row needs 'points'; 'place' defaults to 0 when absent (club/voivodeship rows)
+ */
+function pl_points_rank(array $rows): array
+{
+    usort($rows, function ($a, $b) {
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $a. Configured minimum length is 3. (undefined)

(ShortVariable)